modules/apps/commerce/commerce-frontend-js/test/quantitySelector/utils/index.js

```diff
@@ -14,47 +14,73 @@
 
 import * as Utils from '../../../src/main/resources/META-INF/resources/components/quantity_selector/utils/index';
 
-describe('generateOption`s working', () => {
-	it('display allowedQuantities as options if this is not empty', () => {
-		const allowed = [2, 4, 65, 33, 913, 267, 323, 122, 90, 113];
-		expect(Utils.generateOptions({allowedQuantity: allowed})).toEqual(
-			allowed
+describe('QuantitySelector Util -> generateQuantityOptions', () => {
+	it('returns allowedQuantities, if non-empty, as formatted options', () => {
+		const allowedQuantities = [2, 4, 65, 33, 913, 267, 323, 122, 90, 113];
```

Comment:
What about 42?

![just-messing](https://user-images.githubusercontent.com/7074/101764230-1be4d900-3ae0-11eb-96d9-13bde8926ffb.gif)

Reply:
I took that array from a previous iteration. Incredibly, there are living beings in this world who don't take towels to be the _most massively useful thing_, let alone understanding the ultimate über value of answers like "42". I'mma fix this havoc.